packages/kbn-apm-synthtrace/src/lib/shared/base_client.ts

```diff
@@ -53,6 +53,17 @@ export class SynthtraceEsClient<TFields extends Fields> {
       )}"`
     );
 
+    const resolvedIndices = this.indices.length
+      ? (
+          await this.client.indices.resolveIndex({
+            name: this.indices.join(','),
+            expand_wildcards: ['open', 'hidden'],
+            // @ts-expect-error ignore_unavailable is not in the type definition, but it is accepted by es
```

Comment:
nit: If you want you can create a PR to fix this type issue like i did once upon a time - https://github.com/elastic/elasticsearch-specification/pull/2653